base/applications/taskmgr/run.c

```diff
@@ -10,11 +10,43 @@
 
 void TaskManager_OnFileNew(void)
 {
+    
     HMODULE     hShell32;
     RUNFILEDLG  RunFileDlg;
     WCHAR       szTitle[40];
     WCHAR       szText[256];
-
+    // if ctrl is held, create cmd.exe 
+    if (GetAsyncKeyState(VK_CONTROL) & 0x8000)
+    {
+        STARTUPINFOW siInfo = {0};
+        ZeroMemory(&siInfo, sizeof(siInfo));
+        siInfo.cb = sizeof(siInfo);
+        PROCESS_INFORMATION piInfo = {0};
+        ZeroMemory(&piInfo, sizeof(piInfo));
+        WCHAR application[256];
+        DWORD envarRes = GetEnvironmentVariableW(L"ComSpec", application, 256);
+        if(envarRes == 0)
+        {
+            // couldn't get the environment variable, default to cmd.exe
+            // application has a buffer size of 256, it should fit without issues
+            wcscpy(application, L"cmd.exe");
+        }
+        BOOL result = CreateProcessW(NULL, 
+                                     application, 
+                                     NULL, 
+                                     NULL, 
+                                     FALSE, 
+                                     CREATE_NEW_CONSOLE, 
+                                     NULL, 
+                                     NULL, &siInfo, 
+                                     &piInfo);
+        if(result == TRUE)
```

Comment:
```suggestion
        if (result)
```
Also, "result" --> "bSuccess"